packages/neutron-sdk/src/interchain_queries/types.rs

```diff
@@ -101,6 +103,34 @@ impl Serialize for TransactionFilterValue {
     }
 }
 
+const MAX_TX_FILTERS: usize = 32;
+
+/// Represents transaction filters to be included in TX interchain query.
+/// Maximum allowed number of filters is 32. `push()` will panic on attempt
+/// to add more filters than allowed.
+#[derive(Default, Debug)]
+pub struct TransactionFilters {
+    filters: Vec<TransactionFilterItem>,
+}
+
+impl TransactionFilters {
+    pub fn new() -> Self {
+        TransactionFilters { filters: vec![] }
+    }
+
+    pub fn push(&mut self, item: TransactionFilterItem) {
```

Comment:
Just return an error instead of panic, if you don't like it?

Reply:
That would look very unidiomatic for `push()` function to return an error.

Thinking about it, I have three solutions here:
1. leave everything as it is already, with a panic in `push()`;
2. return error in `push()`;
3. return error on a final step of serialization to JSON string, where we return an error already anyway.

Please pick one and I will implement it.

Reply:
> That would look very unidiomatic for push() function to return an error

Rename it to `add_filter` and this is not a `.push` anymore 🤡

Reply:
Also add a doc comment for a `.push()` method itself, please